### PR TITLE
Update ChangeLog and configure.ac for 1.92

### LINKED ISSUE #2102: Preparing and targeting Release 1.92
I would like to prepare for Release 1.92.

I would like to summarize the bug reports and PRs that are currently blocking this.

I'll try to merge the following PRs early: #2016, #2075, #2097, #2101
There may be other fixes, but we are currently sorting out the issues.

### PR DESCRIPTION
**Pull request by gaul**
Fixes #2102.

### CODE REVIEW

**Comment by gaul** [comment]
Hi @ggtakec, I want to run the next release.  There are always pending PRs but I think this release has a lot of bug fixes that we should give to users.

**Comment by ggtakec** [comment]
I agree to include the unfinished fix in the next release.
Let's go ahead with this release.
Please check the ChangeLog for additional requests.

File: ChangeLog [hunk 1]
```diff
@@ -1,6 +1,18 @@
 ChangeLog for S3FS
 ------------------
 
+Version 1.92 -- 21 May, 2023 (major changes only)
```
Comment (ggtakec) [inline ChangeLog:4]: The stream upload feature will not make it in time for 1.91 and will be included in 1.92, and about credlib, so please add:
```
#1802 - #2104 - New option: streamupload
#1927 - #2101 - New option: credlib and credlib_ops
```

**Review by ggtakec — CHANGES_REQUESTED** [review]
please add two lines to ChangeLog.

File: ChangeLog (hunk 1 above)
Comment (gaul) [reply ChangeLog:4]: Done, sorry I missed the first one and the second one was unclear before.

Comment (ggtakec) [reply ChangeLog:4]: Confirmed the fix.

**Review by ggtakec — APPROVED** [review]
Thanks, I wlll merge this soon.

**Comment by ggtakec** [comment]
@gaul It's merged, so we can create new release tag(v1.92).

**Comment by juliogonzalez** [comment]
As always, here goes my report about submissions to Fedora/EPEL/openSUSE

# Git repository

- PR: https://github.com/juliogonzalez/s3fs-fuse-rpm/pull/27 (merged)
- Release: https://github.com/juliogonzalez/s3fs-fuse-rpm/releases/tag/1.92-1

# openSUSE

- SR to the devel project: https://build.opensuse.org/request/show/1088731
- From there, it should get shortly into openSUSE Tumbleweed/MicroOS, etc.

# Fedora EPEL

Bug: https://bugzilla.redhat.com/show_bug.cgi?id=2209081

All submissions are pending, as I have a running update to adjust the License tag to a valid SPDX expression. Should be released today/tomorrow, and then I should be able to submit 1.92.

**Comment by gaul** [comment]
@juliogonzalez as always thank you for distributing this to the users.  Let me look into Debian and Ubuntu which are still stuck on 1.90.

**Comment by gaul** [comment]
I filed https://bugs.launchpad.net/ubuntu/+source/s3fs-fuse/+bug/2020602 and it seems like the maintainer for Debian is the same so let's wait.

**Comment by juliogonzalez** [comment]
I just completed submissions for EPEL/Fedora. Should be published in 7 days, if there are no issues (exception being Rawhide, which is already published9.

**Pull request merged**